modules/n1ql_fts/pages/index.adoc

```diff
@@ -0,0 +1,589 @@
+= How to use Full-Text Search with N1QL
+
+
+We already discussed how the link:https://blog.couchbase.com/why-you-should-avoid-like-deep-dive-on-fts-part-1/[Like %] clause is potentially harmful for your application at scale, and why you should consider using Couchbase Full-Text Search instead. But one thing we haven't mentioned yet is why you might prefer Couchbase FTS over any other FTS Engine in the market.
+
+Apart from the fact that using Couchbase FTS is simpler in terms of setup and infrastructure, as you can leverage the infrastructure you already have for your database. You can combine the power of N1QL and FTS in a single statement:
+
+
+[source,SQL,indent=0]
+----
+SELECT META(t1).id FROM `travel-sample` AS t1 WHERE SEARCH(t1.country, "United States");
+----
+
+In this tutorial, we will build a simpler version of the application link:https://www.youtube.com/watch?v=B9qRJhA1ONs[built in this talk], which is a basic search engine for movies. 
+
+This application goes much further than simply matching a target text to a field, and can be used as a reference to build production-grade search in your application. Here are some of the key points:
+
+* Best result is returned instead of the best match
+
+* No need to store an indexed field in the FTS index, as we will get the data we need via N1QL.
+
+* User Defined Functions to manipulate mathematically the score according to a given set of rules
+
+
+== Prerequisites
+
+* Couchbase Server 6.5 Beta or Higher
+* Basic understanding of Couchbase and link:https://docs.couchbase.com/server/6.0/fts/full-text-intro.html[Full-Text Search]
+
+== Loading the Dataset
+
+For this demo we will use a dataset of movies exported from IMDB, execute the following steps to import it into your Couchbase instance:
+
+* Create a bucket called "movies"
```

Comment:
I'd prefer numbered items for sequential instructions, so `.` rather than `*`, but be consistent with other tutorials.
Full stop at the end of full sentences throughout this list.

Reply:
I added 1) 2) ...5)  as the counter of **.** restarts after the blocks of code.